proprietary/tokens/README.md

```diff
@@ -1,3 +1,141 @@
+
+The stylesheets wrap the tokens into a `:root` selector.
+If that doesn’t work for your project, use `index.theme.css` instead, which uses an `.ams-theme` selector.
+Add that class to a root element of your application.
+
+### Compact mode
+
+Applications need less whitespace and smaller text sizes than websites.
+Add the compact stylesheet to override the values of these tokens.
+
+```ts
+import "@amsterdam/design-system-tokens/dist/index.css";
+import "@amsterdam/design-system-tokens/dist/compact.css";
+```
+
+Dark mode will work the same way when it’s ready.
+
+### Other formats
+
+The tokens can be imported as Sass variables as well.
+
+```sass
+@import "@amsterdam/design-system-tokens/dist/index.scss"
+```
+
+To use the tokens in JavaScript, import the JSON file.
+
+```ts
+import tokens from "@amsterdam/design-system-tokens/dist/index.json";
+```
+
+## Use the tokens
+
+All tokens' names start with a prefix of `ams-`.
+The name of component tokens ends with the CSS property that uses them, e.g. `-font-size`.
+
+### Component tokens
+
+Every component uses tokens for every property that expresses branding.
+Apply these values in your stylesheets to the corresponding components.
+
+```html
+<button class="my-button" type="button">Button label</button>
+```
+
+```css
+.my-button {
+  font-family: var(--ams-button-font-family);
+  background-color: var(--ams-button-primary-background-color);
+}
+```
+
+### Common tokens
+
+Various groups of components share visual design characteristics.
+E.g. all kinds of links have the same colour, and the borders of various inputs are equally thick.
+We use common tokens to express these relations.
+This also makes it easier to change this shared appearance in the future.
+
+Prefer using common tokens over component tokens.
+If
+
+```html
+<input class="my-input" type="text" />
+```
+
+```css
+.my-input {
+  font-size: var(--ams-common-input-font-size);
+  border-width: var(--ams-common-input-border-width);
+}
+```
+
+### Brand tokens
+
+The brand tokens express the branding of the City of Amsterdam.
+They are our selection from all possible colours, text characteristics, spacing lengths, border widths, etc.
+They are the basis of everything else – no token can have a value for which no brand token exists.
+
+Brand tokens must only be used in common tokens, not in component tokens.
+
+### Overriding tokens
+
+The value of tokens must not be overridden in an application for the City of Amsterdam.
+If you need a different value, create a new token with a different name – and let us know.
+This way, the design system remains consistent and predictable.
+
+A custom theme could restyle our components for a different brand.
+This is an essential feature of [NL Design System](https://nldesignsystem.nl/) of which we are part.
+
+## Support
+
+Contact us if you have a question, find an issue, or want to contribute.
+Find ways to reach us on [designsystem.amsterdam](https://designsystem.amsterdam).
```

Comment:
```suggestion
Find ways to reach us on [designsystem.amsterdam](https://designsystem.amsterdam/?path=/docs/docs-introduction--docs#send-a-message).
```

Reply:
True, but currently it sort of feels like: 'Here's the root url, figure it out yourself'. It's not very user centric, imo. 

I thought we had a ticket to add [SUPPORT.md](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project) to our repo, but I can't find it. Eventually, we should link to that I think, but for now I would prefer the direct link.